body/stretch_body/stepper.py

```diff
@@ -488,7 +488,7 @@ def menu_transaction(self,x,do_print=True):
             while self.transport.ser.inWaiting():
                 r=self.transport.ser.readline()
                 if do_print:
-                    print r,
+                    print(r)
```

Comment:
This is the change I am most unsure about. I'm not sure why the comma was there in the first place, if there was some important reason to not print a newline.